src/components/ContactEdit.tsx

```diff
@@ -309,7 +312,10 @@ class ContactEdit extends React.PureComponent<PropsType> {
     setProperty('title', this.state.title);
     setProperty('note', this.state.note);
 
-    this.props.onSave(contact, this.state.journalUid, this.props.item);
+    this.props.onSave(contact, this.state.journalUid, this.props.item)
```

Comment:
I love it. Looks much nicer! Sorry you had to deal with the mess that was there before.